openedx/core/lib/block_structure/transformers.py

```diff
@@ -1,9 +1,11 @@
 """
 Module for a collection of BlockStructureTransformers.
 """
+from functools import reduce as functools_reduce
```

Comment:
Yeah, it turns out the conflicting [reduce() is a builtin](https://docs.python.org/2/library/functions.html#reduce) which appears to do the same thing. Is there a reason to use the functools version over this?